bin/Indexer.py

```diff
@@ -16,9 +16,28 @@
 from whoosh.index import create_in, exists_in, open_dir
 from whoosh.fields import Schema, TEXT, ID
 import os
+from os.path import join, getsize
 
 from Helper import Process
 
+# Config variable
+TIME_WAIT = 1.0 #sec
+
+# return in bytes
+def check_index_size(baseindexpath, indexname):
+    the_index_name = join(baseindexpath, indexname)
+    cur_sum = 0
+    for root, dirs, files in os.walk(the_index_name):
+        cur_sum += sum(getsize(join(root, name)) for name in files)
+    return cur_sum
+
+def move_index_into_old_index_folder(baseindexpath):
+    command_move = "mv {} {}"
+    os.mkdir(join(baseindexpath, "old_index"))
+    for files in os.listdir(baseindexpath):
+        if not files == "old_index":
+            os.system(command_move.format(join(baseindexpath, files), join(join(baseindexpath, "old_index"), files)))
```

Comment:
It is better to use `os.rename` here: rename https://docs.python.org/2/library/os.html#os.rename 

`os.system` should only be used when the feature cannot be done in python.